### PR TITLE
Add device summary tile to Devices

### PR DESCRIPTION
**Pull request by boaz0**

### CODE REVIEW

File: src/components/AvailableImagesTile.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ import PropTypes from 'prop-types';
 import { useSelector, shallowEqual } from 'react-redux';
 
 const AvailableImageTileBase = ({ children }) => (
-  <Card>
+  <Card style={{ height: '100%' }}>
```
Comment (karelhala) [inline src/components/AvailableImagesTile.js:15]: I don't really like inline styling, would be much better to move this to a SCSS file (either a new one or use `App.scss`)

File: src/components/DeviceSummaryTile.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  StackItem,
+} from '@patternfly/react-core';
+import PropTypes from 'prop-types';
+import { useSelector, shallowEqual } from 'react-redux';
+
+const DeviceSummaryTileBase = ({
+  orphaned,
+  active,
+  no_reports,
+  never_reported,
```
Comment (karelhala) [inline src/components/DeviceSummaryTile.js:19]: No need to use snake case in here.
```suggestion
  noReports,
  neverReported,
```

File: src/components/DeviceSummaryTile.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+        </CardBody>
+      </Card>
+    );
+  }
+  return (
+    <DeviceSummaryTileBase
+      orphaned={54}
+      active={130}
+      no_reports={3}
+      never_reported={31}
```
Comment (karelhala) [inline src/components/DeviceSummaryTile.js:79]: We can use real data for these values, install `@redhat-cloud-services/host-inventory-client` and query for all devices registered with edge for the active section. For now we can add `N/A` for the rest of these values.

File: src/components/Tiles.js [hunk 4]
```diff
@@ -20,9 +21,12 @@ export const Tiles = () => {
   }, [dispatch]);
   return (
     <Flex style={{ paddingBottom: '1rem' }}>
-      <FlexItem>
+      <FlexItem style={{ alignSelf: 'stretch' }}>
```
Comment (karelhala) [inline src/components/Tiles.js:24]: Same as above, this can be moved to SCSS file.

**Comment by boaz0** [comment]
@karelhala I think for some reason `App.scss` is not loaded...

**Comment by karelhala** [comment]
@boaz0 we've changed recently the module name. You'll have to add `  sassPrefix: '.fleet-management',` in the `dev.webpack.config.js` and `prod.webpack.config.js`

**Comment by boaz0** [comment]
That fixed the problem!

https://user-images.githubusercontent.com/2453279/118669261-4741c180-b7fe-11eb-9844-214dd84c00a7.mp4

I opened a new commit for that https://github.com/RedHatInsights/edge-frontend/pull/42

Thanks!

**Comment by boaz0** [comment]
added inventory query field to fetch only edge devices from inventory host
@karelhala mind checking?

File: src/api/index.js [hunk 5]
```diff
@@ -140,3 +146,89 @@ export const updateGroup = ({ uuid, systemIDs, groupName }) => {
...
+      undefined,
+      undefined,
+      undefined,
+      ['fresh'],
+      undefined,
+      undefined,
+      undefined,
+      undefined,
+      {
+        query: generateFilter(inventoryFields, 'fields'),
```
Comment (karelhala) [inline src/api/index.js:169]: No need for the `fields` in here. It should be just the default `filters`
```suggestion
        query: generateFilter(inventoryFields),
```

File: src/api/index.js [hunk 6]
```diff
@@ -140,3 +146,89 @@ export const updateGroup = ({ uuid, systemIDs, groupName }) => {
...
+      undefined,
+      undefined,
+      undefined,
+      ['stale'],
+      undefined,
+      undefined,
+      undefined,
+      undefined,
+      {
+        query: generateFilter(inventoryFields, 'fields'),
```
Comment (karelhala) [inline src/api/index.js:188]:
```suggestion
        query: generateFilter(inventoryFields),
```

File: src/api/index.js [hunk 7]
```diff
@@ -140,3 +146,89 @@ export const updateGroup = ({ uuid, systemIDs, groupName }) => {
...
+      undefined,
+      undefined,
+      undefined,
+      ['stale_warning'],
+      undefined,
+      undefined,
+      undefined,
+      undefined,
+      {
+        query: generateFilter(inventoryFields, 'fields'),
```
Comment (karelhala) [inline src/api/index.js:207]:
```suggestion
        query: generateFilter(inventoryFields),
```

File: src/api/index.js [hunk 8]
```diff
@@ -140,3 +146,89 @@ export const updateGroup = ({ uuid, systemIDs, groupName }) => {
...
+      undefined,
+      undefined,
+      undefined,
+      ['unknown'],
+      undefined,
+      undefined,
+      undefined,
+      undefined,
+      {
+        query: generateFilter(inventoryFields, 'fields'),
```
Comment (karelhala) [inline src/api/index.js:226]:
```suggestion
        query: generateFilter(inventoryFields),
```

**Review by karelhala — APPROVED** [review]

**Pull request merged**